Crashlytics/Crashlytics/DataCollection/FIRCLSDataCollectionArbiter.m

```diff
@@ -20,7 +20,7 @@
 #import "FBLPromises.h"
 #endif
 
-#import "FirebaseCore/Internal/FirebaseCoreInternal.h"
+#import "FirebaseCore/Extension/FirebaseCoreInternal.h"
```

Comment:
Now that module imports are ok, we might want to delete FirebaseCoreInternal.h in favor of `@import FirebaseCoreExtension;`  - although I'm not sure we have a g3 strategy for that and may need to keep FirebaseCoreInternal.h at least on the g3 side?

Reply:
I'm ok to defer this decision.